cmd/kube-apiserver/app/aggregator.go

```diff
@@ -73,6 +74,7 @@ func createAggregatorConfig(
 		&genericConfig,
 		externalInformers,
 		genericConfig.LoopbackClientConfig,
+		feature.DefaultMutableFeatureGate,
```

Comment:
pass in `feature.DefaultFeatureGate`, not the mutable one? (comment applies everywhere)